Guidelines.md

```diff
@@ -1381,13 +1378,13 @@ Accept: application/json
 }
 ```
 
-Service response says the database has been created, but indicates the request is not completed by including the Operation-Location header.
-In this case the status property in the response payload also indicates the operation has not fully completed.
+Service response says the database is being created at the URL provided in the Content-Location header, but indicates the request is not completed by including a 202 status code and Location header pointing to an operation resource.
+The response body includes the incomplete representation of the resource that will eventually exist at the URL in the Content-Location header.
 
 ```http
-HTTP/1.1 201 Created
-Location: https://api.contoso.com/v1.0/databases/db1
-Operation-Location: https://api.contoso.com/v1.0/operations/123
+HTTP/1.1 202 Accepted
+Content-Location: https://api.contoso.com/v1.0/databases/db1
```

Comment:
This is actually **scarier** than a breaking change. You're changing the meaning of the location header, which means developers that aren't paying attention will be actively doing the wrong thing. The interaction of old+new code will be very unpredictable here.

Reply:
If I understand correctly, @darrelmiller is not changing the meaning of the `Location` header. He's pointing out that `201 Created` is a misnomer when we are talking about asynchronous resource creation. A strict interpretation of HTTP would lead one to believe that the resource is done being created. In reality, the create request has been `202 Accepted`, in which case the usage of `Content-Location` is additive to the examples we had before.

Reply:
The meaning of Location header is dependent on the response code.  A 202 response code means the Location header points at the status monitor resource (what we call operation). A 201 status code means the Location points to the created resource.  For 3xx status codes Location means something else.

Therefore because I'm also changing the status code that is returned for a hybrid flow, the meaning of Location header would be consistently the same as what Operation-Location currently means.  Old client code talking to a new API would be missing the Operation-Location header in a 202 response and new client code talking to an old service would see a 201 created and not be aware that it is a hybrid flow LRO.